query-node/mappings/src/content/nft.ts

```diff
@@ -700,6 +706,7 @@ export async function contentNft_AuctionBidCanceled({ event, store }: EventConte
     'Non-existing auction got bid canceled',
     ['topBid', 'bids', 'bids.bidder']
   )
+  const { nft } = video
```

Comment:
Since `video.nft` is not guaranteed to exist, it creates an extra burden of checking it's not null as can be seen a few lines under.
```
ownerMember: nft?.ownerMember,
ownerCuratorGroup: nft?.ownerCuratorGroup,
```

1) `getCurrentAuctionFromVideo` ensures NFT exists, thus it will be better if it returns `nft` object with the other variables it already does (`video` & `auction`).

2) Let's add a check for `nft` existance in `getCurrentAuctionFromVideo` and get rid of optional chaining there.
```
...
const nft = video.nft

if (!nft) {
  return inconsistentState(errorMessageForNft, videoId) # add errorMessageForNft as function parameter
}

// get auction
const allAuctions = video.nft.auctions || []
...
```